### DISCUSSION TITLE
Disable encryption for xrdp connections

### DISCUSSION THREAD

**Discussion by Mogaba** [FreeRDP/FreeRDP#9440] [Q&A] [answered]
Hello,

is it possible to completely disable the encryption in favour of the performance when connecting to xrdp server? I set `crypt_level=none` in xrdp config but when I try to run `xfreerdp -sec-tls -sec-rdp` it throws an error:

`[ERROR][com.freerdp.core.nego] - No security protocol is enabled`

**Comment by akallabeth** [comment, marked as answer]
1. no. and we will not support such things. this opens a can of worms we do not want to touch. (similar why you don´t see `http` these days any more)
2. performance wise this does not have that huge of an impact, check all other areas first! (and specifically for `xrdp` get a branch that supports `gfx` channel before talking about performance ;))
